src/SimplCommerce.WebHost/appsettings.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.\\SQLEXPRESS;Initial Catalog=SimplCommerce;Integrated Security=True;MultipleActiveResultSets=True"
+    "DefaultConnection": "Your ConnectionString"
```

Comment:
I don't think it is a good idea to change this value. It eventually will be overrided.

Reply:
We can make it as its for the novice users, again the `README.md` will describes everything to make the platform up & running

Reply:
SQL Server Express requires a separate install as it doesn't come with the VS installation. For ease if use so new comers can get up and running quickly, please consider MS SQL LocalDB as it comes bundled with VS and it works just as good as SQL Server Express.

Reply:
I agree with what @tdavisjr is suggested, can we change the default connectionString to LocalDB?